docs/src/components/ComponentDoc/ComponentControls/ComponentControls.tsx

```diff
@@ -64,12 +64,10 @@ const ComponentControls: React.FC<ComponentControlsProps> = props => {
   } = props
 
   return (
-    <Provider theme={controlsTheme} as={React.Fragment}>
```

Comment:
Same thing, https://github.com/stardust-ui/react/pull/1290/files#r282797120

Reply:
This change breaks styling, BTW

![image](https://user-images.githubusercontent.com/14183168/57516383-e7fa6e00-7314-11e9-8286-9e3121315fe2.png)